test/mongodb.test.js

```diff
@@ -4,6 +4,36 @@ var should = require('./init.js');
 
 var Superhero, User, Post, PostWithStringId, db;
 
+describe('lazyConnect', function() {
+  it('should skip connect phase (lazyConnect = true)', function(done) {
+    var ds = getDataSource({
+      host: '127.0.0.1',
+      port: 4,
+      lazyConnect: true,
+    });
+    var errTimeout = setTimeout(function() {
+      done();
+    }, 5000);
```

Comment:
This makes the test to run 5 seconds, which is a lot. Can we make the delay smaller please?